plugin/trino-memory/src/main/java/io/trino/plugin/memory/MemoryMetadata.java

```diff
@@ -326,6 +326,20 @@ else if (views.putIfAbsent(viewName, definition) != null) {
         tableIds.put(viewName, nextTableId.getAndIncrement());
     }
 
+    @Override
+    public void setViewComment(ConnectorSession session, SchemaTableName viewName, Optional<String> comment)
```

Comment:
This probably needs to be `synchronized` since `views` is a vanilla hash map and the other implementations also use  `synchronized`.
Would be good to add `@GuardedBy` annotation as well

Reply:
Let me handle `@GuardedBy` in another PR since it unrelates to `COMMENT ON VIEW` change.